models/model.js

```diff
@@ -57,3 +57,34 @@ exports.fetchReviewById = (review_id) => {
       return Promise.reject(err);
     });
 };
+
+exports.provideReviewComment = (review_id, newComment) => {
+  console.log(review_id, newComment);
+
+  if (
+    newComment.hasOwnProperty("username") === false ||
+    newComment.hasOwnProperty("body") === false
+  ) {
+    return Promise.reject({ status: 400, msg: "bad request" });
+  }
+
+  db.query(
+    `INSERT INTO users
+  (username, name)
+  VALUES
+  ($1, $1)RETURNING*;`,
+    [newComment.username]
+  );
```

Comment:
Registering a user would be another process that should be handled separately, and not be automatically done when adding a comment.  For now remove this.

Postgres will throw an error if a non-existing username is trying to add a comment (handle this error).

Change your tests to account for existing/non-existing users.